docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -542,69 +568,115 @@ paths:
+
+            Examples:
+
+            * `http://example.com/articles/01/name` (URL)
+            * `doi:10.1.1/1234` (DOI)
+            * `urn:x-pdf:1234` (PDF fingerprint)
+
           required: false
           type: string
-        - name: uri.parts
-          in: query
-          description: |
-            Limit the results to annotations containing the given keyword in
-            the URL.
         - name: url
           in: query
           description: |
             Alias of `uri`.
+        - name: uri.parts
+          in: query
+          example: 'yogur'
+          description: |
+            <p>Limit the results to annotations containing the given keyword (tokenized chunk) in
+            the URI. The value must exactly match an individual URI keyword.</p>
+
+            <p>URIs are split on characters `#+/:=?.-` into their keywords.</p>
+
+            <p><em>Expanded example </em> — given a value of `yogur`, annotations with any of the
+            following URIs would match:
+
+            * `https://www.yogur.com/foo/bar`
+            * `https://www.example.com/yogur/eatmore.html`
+            * `https://www.example.com/foo/eat-more-yogur-this-year`
+
+            The following would not be matches:
+
+            * `https://www.yogurt.com/foo/bar`
+            * `https://www.example.com/yogurt/eatmore.html`
+            * `https://www.example.com/foo/eat-more-yogurt-this-year`
+
+
           required: false
           type: string
         - name: wildcard_uri
           in: query
+          example: "http://foo.com/*"
           description: |
-            Limit the results to annotations matching the wildcard URI.
-            URI can be a URL (a web page address) or a URN representing another
-            kind of resource such as DOI (Digital Object Identifier) or a
-            PDF fingerprint.
+            <p>Limit the results to annotations whose URIs match the wildcard pattern.</p>
 
-            `*` will match any character sequence (including an empty one),
+            <p>`*` will match any character sequence (including an empty one),
             and a `_` will match any single character. Wildcards are only permitted
-            within the path and query parts of the URI.
+            within the path and query parts of the URI. Escaping wildcards is not supported.</p>
+
+            Examples of valid values:
 
-            Escaping wildcards is not supported.
+            * `http://foo.com/*`
+            * `urn:x-pdf:*`
+            * `file://localhost/_bc.pdf`
 
-            Examples of valid uris: `http://foo.com/*` `urn:x-pdf:*` `file://localhost/_bc.pdf`
+            Examples of invalid values (not within path or query parts of URI):
 
-            Examples of invalid uris: `*foo.com` `u_n:*` `file://*` `http://foo.com*`
+            * `*foo.com`
+            * `u_n:*`
+            * `file://*`
+            * `http://foo.com*`
 
             <mark>This feature is experimental and the API may change.</mark>
```

Comment:
I think we could get rid of this last comment now actually. This was added while we were still developing on it but I think it's solidified now and this could go away.